src/applications/disability-benefits/996/components/IntroductionPage.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+
+import SaveInProgressIntro from '../../../../platform/forms/save-in-progress/SaveInProgressIntro';
+import CallToActionWidget from '../../../../platform/site-wide/cta-widget';
+import { toggleLoginModal } from '../../../../platform/site-wide/user-nav/actions';
+import { focusElement } from '../../../../platform/utilities/ui';
+
+import { WithdrawFromLegacySystem } from './WithdrawFromLegacySystem';
+
+import { BASE_URL } from '../constants';
+
+// import { VerifiedAlert } from '../helpers';
+
+class IntroductionPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    // TODO: get Legacy status; should this value should be in the props.form?
+    // Update this portion once the API has been added
+    this.state = {
+      isInLegacySystem:
+        typeof props?.form?.isInLegacySystem === 'undefined'
+          ? true
+          : props.form.isInLegacySystem,
+    };
+  }
+
+  componentDidMount() {
+    if (
+      !this.hasSavedForm() &&
+      !window.location.pathname.endsWith('/introduction')
+    ) {
+      window.location.replace(`${BASE_URL}/introduction`);
+    }
+    focusElement('.va-nav-breadcrumbs-list');
+  }
+
+  hasSavedForm = () => {
+    const { user } = this.props;
+    return user?.profile?.savedForms
+      .filter(f => moment.unix(f.metadata.expiresAt).isAfter())
+      .find(f => f.form === this.props.formId);
+  };
+
+  authenticate = e => {
+    e.preventDefault();
+    this.props.toggleLoginModal(true);
+  };
+
+  userWithdrewFromLegacySystem = () => {
+    // TODO: replace placeholder; Use API to get/set value; where should the
+    // value set in the this.props.form?
+    this.setState({ isInLegacySystem: false });
+  };
+
+  render() {
+    const { user } = this.props;
+    // console.log(this.props);
```

Comment:
Should probably remove these commented lines once you're done with them.

Reply:
Removed. I also had a reminder checkbox that I added in the `TODO.md` included in this PR (which will also be removed when ready for production).